diff --git a/Makefile b/Makefile
index 9951b7d..6091982 100644
--- a/Makefile
+++ b/Makefile
@@ -5,21 +5,15 @@ PREFIX = /usr/local
 MANDIR = $(PREFIX)/share/man
 
 IN_PROGS = \
-	archco \
 	arch-nspawn \
-	archrelease \
 	archbuild \
 	checkpkg \
-	commitpkg \
-	crossrepomove\
 	finddeps \
 	find-libdeps \
 	lddd \
-	makerepropkg \
 	mkarchroot \
 	makechrootpkg \
 	offload-build \
-	rebuildpkgs \
 	sogrep
 
 BINPROGS = \
@@ -28,47 +22,9 @@ BINPROGS = \
 CONFIGFILES = \
 	makepkg-x86_64.conf \
 	pacman-extra.conf \
-	pacman-testing.conf \
-	pacman-staging.conf \
-	pacman-multilib.conf \
-	pacman-multilib-testing.conf \
-	pacman-multilib-staging.conf \
-	pacman-kde-unstable.conf \
-	pacman-gnome-unstable.conf
 
 SETARCH_ALIASES = \
 
-COMMITPKG_LINKS = \
-	extrapkg \
-	testingpkg \
-	stagingpkg \
-	communitypkg \
-	community-testingpkg \
-	community-stagingpkg \
-	multilibpkg \
-	multilib-testingpkg \
-	multilib-stagingpkg \
-	kde-unstablepkg \
-	gnome-unstablepkg
-
-ARCHBUILD_LINKS = \
-	extra-x86_64-build \
-	testing-x86_64-build \
-	staging-x86_64-build \
-	multilib-build \
-	multilib-testing-build \
-	multilib-staging-build \
-	kde-unstable-x86_64-build \
-	gnome-unstable-x86_64-build
-
-CROSSREPOMOVE_LINKS = \
-	extra2community \
-	community2extra
-
-BASHCOMPLETION_LINKS = \
-	archco \
-	communityco
-
 
 MANS = \
 	doc/archbuild.1 \
@@ -112,14 +68,10 @@ install:
 	install -m0755 ${BINPROGS} $(DESTDIR)$(PREFIX)/bin
 	install -m0644 ${CONFIGFILES} $(DESTDIR)$(PREFIX)/share/devtools
 	for a in ${SETARCH_ALIASES}; do install -m0644 setarch-aliases.d/$$a $(DESTDIR)$(PREFIX)/share/devtools/setarch-aliases.d; done
-	for l in ${COMMITPKG_LINKS}; do ln -sf commitpkg $(DESTDIR)$(PREFIX)/bin/$$l; done
-	for l in ${ARCHBUILD_LINKS}; do ln -sf archbuild $(DESTDIR)$(PREFIX)/bin/$$l; done
-	for l in ${CROSSREPOMOVE_LINKS}; do ln -sf crossrepomove $(DESTDIR)$(PREFIX)/bin/$$l; done
 	ln -sf find-libdeps $(DESTDIR)$(PREFIX)/bin/find-libprovides
 	install -Dm0644 bash_completion $(DESTDIR)/usr/share/bash-completion/completions/devtools
 	for l in ${BASHCOMPLETION_LINKS}; do ln -sf devtools $(DESTDIR)/usr/share/bash-completion/completions/$$l; done
 	install -Dm0644 zsh_completion $(DESTDIR)$(PREFIX)/share/zsh/site-functions/_devtools
-	ln -sf archco $(DESTDIR)$(PREFIX)/bin/communityco
 	for manfile in $(MANS); do \
 		install -Dm644 $$manfile -t $(DESTDIR)$(MANDIR)/man$${manfile##*.}; \
 	done;
@@ -128,12 +80,8 @@ uninstall:
 	for f in ${BINPROGS}; do rm -f $(DESTDIR)$(PREFIX)/bin/$$f; done
 	for f in ${CONFIGFILES}; do rm -f $(DESTDIR)$(PREFIX)/share/devtools/$$f; done
 	for f in ${SETARCH_ALIASES}; do rm -f $(DESTDIR)$(PREFIX)/share/devtools/setarch-aliases.d/$$f; done
-	for l in ${COMMITPKG_LINKS}; do rm -f $(DESTDIR)$(PREFIX)/bin/$$l; done
-	for l in ${ARCHBUILD_LINKS}; do rm -f $(DESTDIR)$(PREFIX)/bin/$$l; done
-	for l in ${CROSSREPOMOVE_LINKS}; do rm -f $(DESTDIR)$(PREFIX)/bin/$$l; done
 	rm $(DESTDIR)/usr/share/bash-completion/completions/devtools
 	rm $(DESTDIR)$(PREFIX)/share/zsh/site-functions/_devtools
-	rm -f $(DESTDIR)$(PREFIX)/bin/communityco
 	rm -f $(DESTDIR)$(PREFIX)/bin/find-libprovides
 	for manfile in $(MANS); do \
 		rm -f $(DESTDIR)$(MANDIR)/man$${manfile##*.}/$${manfile#doc/}; \
