From d190403a428a1efeca6cb3c1d8c2a00093e91754 Mon Sep 17 00:00:00 2001
From: tinywrkb <tinywrkb@gmail.com>
Date: Wed, 30 Jun 2021 11:33:29 +0300
Subject: [PATCH] time-util: recursive localtime resolution

---
 src/basic/time-util.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/basic/time-util.c b/src/basic/time-util.c
index 3c2b25bd2a..d8d12984dd 100644
--- a/src/basic/time-util.c
+++ b/src/basic/time-util.c
@@ -1455,7 +1455,7 @@ int get_timezone(char **ret) {
         char *z;
         int r;
 
-        r = readlink_malloc("/etc/localtime", &t);
+        r = chase_symlinks("/etc/localtime", NULL, NULL, &t, NULL);
         if (r == -ENOENT) {
                 /* If the symlink does not exist, assume "UTC", like glibc does*/
                 z = strdup("UTC");
-- 
2.32.0

