From 2d9916579b3816630b16cd04c44ab6fec0d91706 Mon Sep 17 00:00:00 2001
From: tinywrkb <tinywrkb@gmail.com>
Date: Sun, 13 Sep 2020 19:42:24 +0300
Subject: [PATCH] no more /run bind mount

---
 common | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/common b/common
index fa040e7..baa581b 100644
--- a/common
+++ b/common
@@ -89,7 +89,10 @@ chroot_setup() {
   chroot_add_mount udev "$1/dev" -t devtmpfs -o mode=0755,nosuid &&
   chroot_add_mount devpts "$1/dev/pts" -t devpts -o mode=0620,gid=5,nosuid,noexec &&
   chroot_add_mount shm "$1/dev/shm" -t tmpfs -o mode=1777,nosuid,nodev &&
-  chroot_add_mount /run "$1/run" --bind &&
+  chroot_add_mount run "$1/run" -t tmpfs -o nosuid,nodev,mode=0755 &&
+  mkdir "$1/run/lvm" &&
+  ignore_error chroot_maybe_add_mount "[[ -d '/run/lvm' ]]" \
+      /run/lvm "$1/run/lvm" --bind &&
   chroot_add_mount tmp "$1/tmp" -t tmpfs -o mode=1777,strictatime,nodev,nosuid
 }
 
-- 
2.28.0

